### ISSUE TITLE
Mobile: Content display shrinks on mobile devices making it not readable.

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Residential-Building-Brokers#12]
![image](https://cloud.githubusercontent.com/assets/12791909/11432927/06c6edc2-94f2-11e5-9416-047a1d7a1d1f.png)

**Comment by titanguest** [comment]
This is per Bernard's request, as it is need to be raised to Jolene for confirmation.

**Comment by ermatitan** [comment]
A confirmation is needed before we can close the issue. Thanks

**Comment by ermatitan** [comment]
Won't fix issue as confirmed by Bernard Salas
